### PR TITLE
[quest] Fix 'Cleansing Our Crevasse' Objectives

### LINKED ISSUE #6204: Quest Issue for quest 14381 - Cleansing Our Crevasse (Desolace, Cataclysm Classic)
## Description
Quest does not display the required mobs on the map, or indicate on the mob tooltip that they are relevant for the quest

## Wowhead link
Quest: https://www.wowhead.com/cata/quest=14381/cleansing-our-crevasse
Mobs: https://www.wowhead.com/cata/npc=35606/slitherblade-siren
https://www.wowhead.com/cata/npc=35605/slitherblade-invader
https://www.wowhead.com/cata/npc=90/sea-giant

## Questie & WoW version
Questie v10.3.7 & Classic Cataclysm

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #6204

## Proposed changes

- Add a fix to `objectives` for the 'Cleansing Our Crevasse' quest.

### CODE REVIEW

File: Database/Corrections/cataQuestFixes.lua [hunk 1]
```diff
@@ -1727,6 +1727,9 @@ function CataQuestFixes.Load()
             [questKeys.preQuestGroup] = {14370,14371},
             [questKeys.objectives] = {{{36297,nil,Questie.ICON_TYPE_TALK}}},
         },
+        [14381] = { -- Cleansing Our Crevasse
+            [questKeys.objectives] = {{{90},{35605},{35606}}},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:1731]: This is not correct.

The quest has 4 "monster" objectives as per wowhead: https://www.wowhead.com/cata/quest=14381/cleansing-our-crevasse (which can be trusted in 99% of the cases).

Also the order matters.

**Review by BreakBB — CHANGES_REQUESTED** [review]

File: Database/Corrections/cataQuestFixes.lua (hunk 1 above)
Comment (Sommos) [reply Database/Corrections/cataQuestFixes.lua]: https://github.com/Questie/Questie/pull/6211/commits/1104054bdb0712e05a6d93b448266a6496115d83

File: Database/Corrections/cataQuestFixes.lua [hunk 2]
```diff
@@ -1754,6 +1754,9 @@ function CataQuestFixes.Load()
         [14379] = { -- Rock Lobstrock!
             [questKeys.preQuestGroup] = {14373,14374},
         },
+        [14381] = { -- Cleansing Our Crevasse
+            [questKeys.objectives] = {{{36232,nil,Questie.ICON_TYPE_MOUNT_UP},{36228},{36230},{36229}}},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:1758]: Now you are only copying what the database already have and change the icon for the first NPC. The last three objectives are "kill credit bunny" objectives. Meaning while they are marked as NPCs in the database, they are no real NPCs (blizzard does that every now and then for a quest). You need to find the actual NPCs to replace them.

**Review by BreakBB — CHANGES_REQUESTED** [review]

File: Database/Corrections/cataQuestFixes.lua (hunk 2 above)
Comment (Sommos) [reply Database/Corrections/cataQuestFixes.lua]: https://github.com/Questie/Questie/pull/6211/commits/0cdd91e9b34fbc7bb13891bd9c4191f5918dce99

Should fix these issues. The first NPC needs to have a `ICON_TYPE_MOUNT_UP` as the player needs to ride Grimfeather to do the other objectives. The bunnys have been switched with true NPC IDs.

**Pull request merged**